packit/config/common_package_config.py

```diff
@@ -94,11 +106,34 @@ def __init__(
         self.sources = sources or []
         self.merge_pr_in_ci = merge_pr_in_ci
 
+    @property
+    def files_to_sync(self) -> List[SyncFilesItem]:
+        """
+        synced_files is the old option we want to deprecate.
+        Spec file and configuration file can be automatically added to
+        the list of synced_files (see get_all_files_to_sync method)
+
+        files_to_sync is the new option. Files to be synced are just those listed here.
+        Spec file and configuration file will not be automatically added
+        to the list of files_to_sync (see get_all_files_to_sync method).
+
+        files_to_sync has precedence over synced_files.
+
+        Once the old option will be removed this method can be removed as well.
+        """
+
+        if self._files_to_sync:
+            return self._files_to_sync
+        elif self._synced_files:
+            return self._synced_files
```

Comment:
I'm thinking we should let the user know when both are being set. A warning "You are setting both files_to_sync and synced_file: Packit will use files_to_sync, so you should remove synced_files since it's deprecated." would explain to the user what's happening.

Reply:
I am already writing a warning on the console every time an old synced_files option is found, I am doing it in the __init__method of CommonPackageConfig. Do you want me to move it here?
The message is "synced_files option is deprecated. Use files_to_sync option instead."

The annoying thing is that this message is shown 3 times when you do something with CLI but I thought this was not a problem.

Reply:
oki, makes sense to me; but please let's document the behaviour of files_to_sync having a bigger priority than sync_files

3 times? that will motivate people to change :)

Reply:
> oki, makes sense to me; but please let's document the behaviour of files_to_sync having a bigger priority than sync_files

Yes sure